ops.yml

```
@@ -77,3 +77,4 @@ actions:
 options:
   environment:
     INSTALL_DIR: "$HOME/bin/"
+    CRYSTAL_PATH: "`crystal env CRYSTAL_PATH`:./src"
```

Should `src` come before `CRYSTAL_PATH`?

---

```suggestion
    CRYSTAL_PATH: "./src:`crystal env CRYSTAL_PATH`"
```
Weird, I actually remember having `src` before not after...